introduction.md

```diff
@@ -1 +1,3 @@
 Hello! I am Muyuan Ma and I am learning about GitHub from Code School. My favorite Code School path is Ruby.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?